docs/pages/mydoc/crdts.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Datatypes in Antidote
+tags: [crdts]
+sidebar: mydoc_sidebar
+permalink: crdts.html
+folder: mydoc
+---
+
+
+Most highly available databases only provide last-writer-wins registers, which resolve concurrent updates using timestamps.
+Antidote offers a variety of other datatypes with smarter strategies to resolve concurrent updates.
+The available data types are described below.
+
+All data types have in common, that their value only depends on the operations which have been delivered to the currently used Antidote data center.
+For all datatypes there can be concurrent operations which are not yet visible.
```

Comment:
When writing I did not think of the state but of the currently executing operation or the current transaction and since this is not complete yet, there can be no operations after it.

When thinking about the state I see that my sentence is not correct. I'll try to reformulate it.